contrib/index.html

```diff
@@ -0,0 +1,199 @@
+  <!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <meta charset="utf-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+      <meta name="viewport" content="width=device-width,initial-scale=1">
+      <title>Have fun ♥️</title>
```

Comment:
It's just…
<img width="236" alt="screen shot 2015-11-27 at 4 18 22 pm" src="https://cloud.githubusercontent.com/assets/2489247/11448259/87bdd212-9522-11e5-809a-420843ee0c64.png">